src/plumbing/transcoding.c

```diff
@@ -540,13 +540,16 @@ transcoder_stream_audio(transcoder_t *t, transcoder_stream_t *ts, th_pkt_t *pkt)
       }
     }
 
+    // User specified streaming profile audio bitrate limiter.
+    if (t->t_props.tp_abitrate >=16) {
```

Comment:
Could you review your changes and put spaces between condition and numbers like '>= 16' ?